lib/Espfc/src/Device/InputESPNOW.h

```diff
@@ -0,0 +1,187 @@
+#ifndef _INPUT_DEVICE_INPUT_ESPNOW_H_
+#define _INPUT_DEVICE_INPUT_ESPNOW_H_
+
+#include "esp_wifi.h"
+#include <WiFi.h>
+#include <esp_now.h>
+#include "Device/InputDevice.h"
+#include "Math/Utils.h"
+
+
+
+namespace Espfc {
+
+namespace Device {
+
+struct ControllerData {
+  uint8_t syncByte: 1;
+  uint16_t A: 10;
+  uint16_t E: 10;
+  uint16_t R: 10;
+  uint16_t T: 10;
+
+  uint8_t ch1;
+  uint8_t ch2;
+  uint8_t ch3;
+  uint8_t ch4;
+}_data;
+
+int8_t rssi;
+
+struct TelemetryData {
+  uint8_t syncByte: 1;
+
+  uint8_t Battery_v; // int/10 -> fixed point 25.6v max
+}TxData;
+
+//unsigned long lastRecvTime = 0;
+bool _new_data = false;
```

Comment:
avoid using global variables as much as possible, it would be ideal if `_data`, `TxData` and `_new_data` were members of the class.